crates/devolutions-agent-shared/src/lib.rs

```diff
@@ -1,6 +1,10 @@
 mod date_version;
 mod update_json;
+#[cfg(windows)]
+pub mod windows;
```

Comment:
suggestion: We likely don’t need this to be public anymore.
```suggestion
#[cfg(windows)]
mod windows;
```

Reply:
We still need this to be public as the Agent uses product codes to invoke msiexec, and we can't make MSI product codes cross-platform in shared library, lets keep it as-is for now